src/main/java/digital/vix/vertxdemo/controller/InformationController.java

```diff
@@ -0,0 +1,86 @@
+package digital.vix.vertxdemo.controller;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import digital.vix.vertxdemo.models.Information;
+import digital.vix.vertxdemo.service.InformationService;
+import io.reactivex.Completable;
+import io.vertx.reactivex.core.AbstractVerticle;
+import io.vertx.reactivex.ext.web.Router;
+import io.vertx.reactivex.ext.web.RoutingContext;
+import io.vertx.reactivex.ext.web.handler.BodyHandler;
+
+public class InformationController extends AbstractVerticle {
+
+	private Router router;
+	private ObjectMapper mapper;
+	private InformationService informationService;
+
+	public InformationController(Router router, ObjectMapper mapper, InformationService informationService) {
+		this.router = router;
+		this.mapper = mapper;
+		this.informationService = informationService;
+	}
+
+	@Override
+	public Completable rxStart() {
+		router.get("/api/information").handler(this::get);
+		router.post("/api/information").handler(BodyHandler.create()).handler(this::add);
+		router.put("/api/information").handler(BodyHandler.create()).handler(this::update);
+		router.delete("/api/information/:id").handler(this::delete);
+		return informationService.all().map(jsonObject -> {
+			Information information = mapper.readValue(jsonObject.toString(), Information.class);
+			return information;
+		}).ignoreElements();
+	}
```

Comment:
This doesn't need to be an rxStart because the information we get back is instantly ignored and not really used anywhere... 

```suggestion
	@Override
	public void start() {
		router.get("/api/information").handler(this::get);
		router.post("/api/information").handler(BodyHandler.create()).handler(this::add);
		router.put("/api/information").handler(BodyHandler.create()).handler(this::update);
		router.delete("/api/information/:id").handler(this::delete);
	}
```